models/polls.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  name: "polls",
+  schema: {
+    name: String,
+    creator: String,
+    createdBy: {
+      type: String,
+      required: true,
+      index: true
+    },
+    questions: Array,
+    link: {
+      type: String,
+      index: true
+    },
+    result: Object,
+    numberOfResponses: Number,
+    metaInformation: Object,
+    isDeleted: {
+      type: Boolean,
+      default: false
+    },
+    startDate: Date,
```

Comment:
@deepakr123 - Make fields which will always be there as mandatory.

Reply:
resolved